### ISSUE TITLE
Latest beta bailing when loading rules

### ISSUE THREAD

**Issue by richardtallent** [vuejs/eslint-plugin-vue#288]
**Tell us about your environment**
- OS X (High Sierra)
- Brand new app using Vuetify's PWA template (based on the Vue PWA template)
- Adjusted package.json to update to latest versions

* **ESLint Version:** 4.12.0
* **eslint-plugin-vue Version:** eslint-plugin-vue@4.0.0-beta.2
* **Node Version:** 6.9.2

**Please show your full configuration:**

.eslintrc.js
```js
module.exports = {
  root: true,
  parser: 'babel-eslint',
  parserOptions: {
    sourceType: 'module'
  },
  env: {
	browser: true,
	es6: true
  },
  globals: {
	"Vue": true
  },
  extends: [
	'eslint:recommended',
	'plugin:vue/essential'
	],
  // required to lint *.vue files
  plugins: [
    'html'
  ],
  // check if imports actually resolve
  'settings': {
    'import/resolver': {
      'webpack': {
        'config': 'build/webpack.base.conf.js'
      }
    }
  },
  // add your custom rules here
  'rules': {
	'vue/html-indent': 'tab',
	'vue/max-attributes-per-line': 3,
	// don't require .vue extension when importing
    'import/extensions': ['error', 'always', {
      'js': 'never',
      'vue': 'never'
    }],
    // allow optionalDependencies
    'import/no-extraneous-dependencies': ['error', {
      'optionalDependencies': ['test/unit/index.js']
    }],
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0
  }
}
```

package.json:
```json
{
  "private": true,
  "scripts": {
    "dev": "node build/dev-server.js",
    "start": "node build/dev-server.js",
    "build": "node build/build.js",
    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
    "e2e": "node test/e2e/runner.js",
    "test": "npm run unit && npm run e2e",
    "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs"
  },
  "dependencies": {
    "vue": "^2.5.9",
    "vue-router": "^3.0.1",
    "vuetify": "^0.17.4"
  },
  "devDependencies": {
    "autoprefixer": "^7.2.2",
    "babel-core": "^6.26.0",
    "babel-eslint": "^8.0.3",
    "babel-loader": "^7.1.2",
    "babel-plugin-istanbul": "^4.1.5",
    "babel-plugin-transform-runtime": "^6.23.0",
    "babel-polyfill": "^6.26.0",
    "babel-preset-env": "^1.6.1",
    "babel-preset-stage-2": "^6.24.1",
    "babel-register": "^6.26.0",
    "chai": "^4.1.2",
    "chalk": "^2.3.0",
    "chromedriver": "^2.33.2",
    "connect-history-api-fallback": "^1.5.0",
    "copy-webpack-plugin": "^4.2.3",
    "cross-env": "^5.1.1",
    "cross-spawn": "^5.1.0",
    "css-loader": "^0.28.7",
    "cssnano": "^3.10.0",
    "eslint": "^4.12.0",
    "eslint-friendly-formatter": "^3.0.0",
    "eslint-import-resolver-webpack": "^0.8.3",
    "eslint-loader": "^1.9.0",
    "eslint-plugin-html": "^4.0.1",
    "eslint-plugin-import": "^2.8.0",
    "eslint-plugin-vue": "next",
    "eventsource-polyfill": "^0.9.6",
    "express": "^4.16.2",
    "extract-text-webpack-plugin": "^3.0.2",
    "file-loader": "^1.1.5",
    "friendly-errors-webpack-plugin": "^1.6.1",
    "html-webpack-plugin": "^2.30.1",
    "http-proxy-middleware": "^0.17.4",
    "inject-loader": "^3.0.1",
    "karma": "^1.7.1",
    "karma-coverage": "^1.1.1",
    "karma-mocha": "^1.3.0",
    "karma-phantomjs-launcher": "^1.0.4",
    "karma-phantomjs-shim": "^1.5.0",
    "karma-sinon-chai": "^1.3.3",
    "karma-sourcemap-loader": "^0.3.7",
    "karma-spec-reporter": "0.0.31",
    "karma-webpack": "^2.0.6",
    "lolex": "^2.3.1",
    "mocha": "^4.0.1",
    "nightwatch": "^0.9.19",
    "opn": "^5.1.0",
    "optimize-css-assets-webpack-plugin": "^3.2.0",
    "ora": "^1.3.0",
    "phantomjs-prebuilt": "^2.1.16",
    "postcss-sass": "^0.2.0",
    "rimraf": "^2.6.2",
    "selenium-server": "^3.8.1",
    "semver": "^5.4.1",
    "shelljs": "^0.7.8",
    "sinon": "^4.1.3",
    "sinon-chai": "^2.14.0",
    "stylelint": "^8.3.1",
    "stylelint-config-standard": "^18.0.0",
    "stylus": "^0.54.5",
    "stylus-loader": "^3.0.1",
    "sw-precache-webpack-plugin": "^0.11.4",
    "uglify-es": "^3.2.1",
    "url-loader": "^0.6.2",
    "vue-loader": "^13.5.0",
    "vue-style-loader": "^3.0.3",
    "vue-template-compiler": "^2.5.9",
    "webpack": "^3.10.0",
    "webpack-bundle-analyzer": "^2.9.1",
    "webpack-dev-middleware": "^1.12.2",
    "webpack-hot-middleware": "^2.21.0",
    "webpack-merge": "^4.1.1"
  },
  "engines": {
    "node": ">= 4.0.0",
    "npm": ">= 3.0.0"
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not ie <= 10"
  ]
}
```

**What did you do? Please include the actual source code causing the issue.**

```
npm run dev
```

**What did you expect to happen?**

I expected my new web app to start up. It's basically the latest Vuetify PWA template with various packages updated.

**What actually happened? Please include the actual, raw output from ESLint.**

```
ERROR  Failed to compile with 1 errors                                                                      11:51:18 AM

Module build failed: TypeError: Error while loading rule 'vue/no-duplicate-attributes': Cannot convert undefined or null to object
    at Object.keys.forEach.ex (/foo/node_modules/eslint/lib/linter.js:928:24)
    at Array.forEach (native)
    at Linter._verifyWithoutProcessors (/foo/node_modules/eslint/lib/linter.js:865:35)
    at preprocess.map.textBlock (/foo/node_modules/eslint/lib/linter.js:994:35)
    at Array.map (native)
    at Linter.verify (/foo/node_modules/eslint/lib/linter.js:993:42)
    at localVerify (/foo/node_modules/eslint-plugin-html/src/index.js:81:14)
    at Linter.verify (/foo/node_modules/eslint-plugin-html/src/index.js:145:18)
    at Linter.verifyAndFix (/foo/node_modules/eslint/lib/linter.js:1085:29)
    at processText (/foo/node_modules/eslint/lib/cli-engine.js:179:32)
    at CLIEngine.executeOnText (/foo/node_modules/eslint/lib/cli-engine.js:614:17)
    at lint (/foo/node_modules/eslint-loader/index.js:218:17)
    at Object.module.exports (/foo/node_modules/eslint-loader/index.js:213:21)

You may use special comments to disable some warnings.
Use // eslint-disable-next-line to ignore the next line.
Use /* eslint-disable */ to ignore all warnings in a file.
> Listening at http://localhost:8080
```

I started with the "recommended" rule list and received an identical error, but for another rule. I thought it might be an issue with the rule, so I changed the config to the essential list.

**Comment by michalsnik** [comment]
Hey @richardtallent , please remove `eslint-plugin-html` package and `plugins` section from your `.eslintrc.js`. You can find working `eslint` configuration here: https://github.com/michalsnik/vue-template/blob/master/template/.eslintrc.js

**Comment by richardtallent** [comment]
Thanks! That seems to have done the trick.
